flexget/components/emby/api_emby.py

```diff
@@ -2705,11 +2709,22 @@ def strtotime(date) -> 'datetime':
         elif not isinstance(date, str):
             return None
 
-        # Normalize date
-        date_py = re.sub(r'^(.*)\.([0-9]{6})[0-9]*\+([0-9]{2})\:([0-9]{2})$', r'\1.\2+\3\4', date)
+        # Remove the 'Z' character if present
+        date = date.replace('Z', '')
+
+        # Check if the offset is present
+        if '+' in date:
+            date = date.split('+')[0]
+
+        # Check if ms is present
+        if '.' in date:
+            date = date.split('.')[0]
+
+        # Define the format
+        format_str = '%Y-%m-%dT%H:%M:%S'
```

Comment:
Do we need all this custom date wrangling? Seems like it's probably just an iso date and we can use https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.isoparse

Reply:
I have found that the api is not very accurate returning the date format, I have found more that one format, and had some errors because of that, so i tried to decode the formats and act accordingly! 

https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.isoparse seams to map all the versions! I can try to replace it!

I have other improvements to the lookup! Can I push them in this PR? Since it's related. I've been making some tuning to the lookup, to help getting the matching more accurate.